src/environment.c

```diff
@@ -0,0 +1,369 @@
+#include <err.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+
+#include "bolos/cx.h"
+#include "bolos/endorsement.h"
+#include "bolos/exception.h"
+#include "emulate.h"
+#include "environment.h"
+
+/* SEED VARIABLES */
+
+/* glory promote mansion idle axis finger extra february uncover one trip
+ * resource lawn turtle enact monster seven myth punch hobby comfort wild raise
+ * skin */
+static uint8_t default_seed[MAX_SEED_SIZE] =
+    "\xb1\x19\x97\xfa\xff\x42\x0a\x33\x1b\xb4\xa4\xff\xdc\x8b\xdc\x8b\xa7\xc0"
+    "\x17\x32\xa9\x9a\x30\xd8\x3d\xbb\xeb\xd4\x69\x66\x6c\x84\xb4\x7d\x09\xd3"
+    "\xf5\xf4\x72\xb3\xb9\x38\x4a\xc6\x34\xbe\xba\x2a\x44\x0b\xa3\x6e\xc7\x66"
+    "\x11\x44\x13\x2f\x35\xe2\x06\x87\x35\x64";
+
+static const char *SEED_ENV_NAME = "SPECULOS_SEED";
```

Comment:
I would have expect a `#define` or a `static const char xx[]` but I guess it works.

Reply:
I could go with it if it's more idiomatic.

Reply:
https://stackoverflow.com/a/1431607 :man_shrugging:

Reply:
https://github.com/LedgerHQ/speculos/pull/440/commits/1c47dccaf473278a133ecdc63b7d34a3828cd1c4